deployment/v3/external/kafka/README.md

```diff
@@ -8,6 +8,89 @@
 $ ./install.sh
 ```
 
+## Steps to Backup & Restore Bitnami Kafka manually
+
+#### Prerequisites
+* Install `zip` & `unzip` packages on both backup/restore control plane nodes.
+
+#### Backup
+* Set WebSub and Kafka replicas to zero via the below command.
+  ```
+  kc2 -n default scale --replicas=0 deploy consolidator-websub-service websub-service
+  kc2 -n default scale --replicas=0 statefulset kafka kafka-zookeeper
+  ```
+* Once Websub & Kafka pods are completely terminated, go to `/srv/nfs/mosip/`.
+  ```
+  cd /srv/nfs/mosip
+  ```
+* Run the below command to zip DMZ Kafka directories.
+  ```
+  sudo zip -r dmzkafka-backup.zip $( ls -d $( kc2 get pv | awk '/kafka/{print "*" $1}'))
+  ```
+* Move the backup file to the machine where the restore cluster is present.
+* Set the Kafka replicas to their original value via the below command.
+  ```
+  kc2 -n default scale --replicas=<no-of-replicas> statefulset kafka kafka-zookeeper
+  ```
+* Set the websub replicas to their original value via the below command.
+  ```
+  kc2 -n default scale --replicas= deploy consolidator-websub-service websub-service
```

Comment:
Since we had stopped websub and then starting the same now, lets add a line to restart all the websub dependent services.